### PR TITLE
Feat: added login and signup page

### PR DESCRIPTION
**Pull request by AshwiniSetty**
This PR contains:
1. Login Page
2. Signup Page

image

image

Preview:
https://deploy-preview-8--minwii-stores.netlify.app/

### CODE REVIEW

File: css/login_page.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.login-form{
+    background-color: var(--white);
+    padding: 3rem;
+    width: 400px;
+}
+
+.form-input h2, h4{
+    text-align: center;
+margin-bottom: 1.2rem;
+}
```
Comment (Prashantrajput11) [inline css/login_page.css]: Just a minor suggestion : the code formatting could be improved here

Comment (AshwiniSetty) [reply css/login_page.css]: Yes, will do that.

File: pages/login_page.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width>, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/dd0d05bba7.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Minwii | Wishlist</title>
```
Comment (Prashantrajput11) [inline pages/login_page.html]: I think by mistake you added the wrong title here for the login page.
Rest code looks good. Liked the use of semantic html  and consistent naming

Comment (AshwiniSetty) [reply pages/login_page.html]: Yes, rectified it. Thank you

**Review by Prashantrajput11 — APPROVED** [review]

**Comment by JaswanthReddyPadala** [comment]
Do the changes mentioned in above comments.
Btw, UI is simple and clean.
GTG 🚀

**Review by JaswanthReddyPadala — APPROVED** [review]

**Review by khushijohri2001 — COMMENTED** [review]
In sign up, you should add fields like First Name, Last name, and Confirm Password too. I think it was mandatory to add and they might ask for resubmission if not added. Othewise the code looks fine.

**Pull request merged**